windows/utils/win_disable_bitlocker.yml

```diff
@@ -0,0 +1,72 @@
+# Copyright 2022-2023 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Disable BitLocker service and decrypt the volumes
+# Parameters:
+#   decrypt_wait_time: the time in seconds to wait for the volume decryption.
+#
+- name: "Initialize the decryption wait time"
+  ansible.builtin.set_fact:
+    decrypt_wait_time: 900
+  when: decrypt_wait_time is undefined or not decrypt_wait_time
```

Comment:
The initialization can be moved into block of "Decrypt the BitLocker volumes"

Reply:
Fixed.